components/styles.js

```diff
@@ -49,6 +49,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     padding: 5,
     width: DEVICE_WIDTH / 7,
+    height: DEVICE_WIDTH / 7,
```

Comment:
Agreed. Maybe then I would set the height of the week row to default to `DEVICE_WIDTH / 7` as the solution for this was calculate the height of the whole calendar based on height of the row.